internal/iam/command_ipfilter_list.go

```diff
@@ -1 +1,64 @@
 package iam
+
+import (
+	iamv2 "github.com/confluentinc/ccloud-sdk-go-v2/iam/v2"
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+func (c *ipFilterCommand) newListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List IP filters.",
+		Args:  cobra.NoArgs,
+		RunE:  c.list,
+	}
+
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *ipFilterCommand) list(cmd *cobra.Command, _ []string) error {
+
+	ipFilters, err := c.V2Client.ListIamIpFilters()
+	if err != nil {
+		return err
+	}
+
+	if output.GetFormat(cmd) == output.Human {
+		list := output.NewList(cmd)
```

Comment:
We should only have to define this once.

Reply:
What should only be defined once? The only reused code here is:
```
		list := output.NewList(cmd)
		for _, filter := range ipFilters {
			ipGroupIds := convertIpGroupObjectsToIpGroupIds(filter)
```
It didn't make sense to stick those lines in their own separate method, since they each have a unique `list.add()` buried inside of the for loop.

Reply:
`output.NewList()` is called in two separate places, but it can be moved out of the `if`. Likewise, `list.Print()` can also be moved outside of the `if`.